distribution/src/main/packaging/scripts/prerm

```diff
@@ -79,12 +79,6 @@ if [ "$REMOVE_SERVICE" = "true" ]; then
     if command -v update-rc.d >/dev/null; then
         update-rc.d elasticsearch remove >/dev/null || true
     fi
-
-    SCRIPTS_DIR="/etc/elasticsearch/scripts"
```

Comment:
This is for package removal. If a user is removing an old version, it is the package removal script from the version that they have installed that will execute. We can safely remove this here.